packages/react-input/src/components/Input/useInputStyles.ts

```diff
@@ -1,28 +1,225 @@
+import type { InputState } from './Input.types';
+import type { Theme } from '@fluentui/react-theme';
 
-/**
- * Styles for the root slot
- */
-const useStyles = makeStyles<keyof InputSlots | 'root'>({
-  root: theme => ({}),
-  input: theme => ({}),
-  inputWrapper: theme => ({}),
-  bookendBefore: theme => ({}),
-  bookendAfter: theme => ({}),
-  insideStart: theme => ({}),
-  insideEnd: theme => ({}),
+// TODO(sharing) use theme values once available
+const horizontalSpacing = {
+  xxs: '2px',
+  xs: '4px',
+  sNudge: '6px',
+  s: '8px',
+  mNudge: '10px',
+  m: '12px',
+};
+const contentSizes = {
+  // TODO(sharing) shouldn't these be in the theme?
+  body1: (theme: Theme) => ({
+    fontSize: theme.global.type.fontSizes.base[300],
+    lineHeight: theme.global.type.lineHeights.base[300],
+  }),
+  caption1: (theme: Theme) => ({
+    fontSize: theme.global.type.fontSizes.base[200],
+    lineHeight: theme.global.type.lineHeights.base[200],
+  }),
+  // eslint-disable-next-line @typescript-eslint/naming-convention
+  400: (theme: Theme) => ({
+    fontSize: theme.global.type.fontSizes.base[400],
+    lineHeight: theme.global.type.lineHeights.base[400],
+  }),
+};
+// TODO(sharing) should these be shared somewhere?
+const fieldHeights = {
+  small: '24px',
+  medium: '32px',
+  large: '40px',
+};
+const borderRadius = (theme: Theme) => theme.global.borderRadius.medium;
+const backgroundColors = {
+  /** for outline/filledLighter */
+  filledLighter: (theme: Theme) => theme.alias.color.neutral.neutralBackground1,
+  filledDarker: (theme: Theme) => theme.alias.color.neutral.neutralBackground3,
+  /** for underline */
+  transparent: (theme: Theme) => theme.alias.color.neutral.transparentBackground,
+};
+
+const useStyles = makeStyles({
+  root: theme => ({
+    display: 'flex',
+    alignItems: 'stretch',
+    minHeight: fieldHeights.medium, // minHeight not height in case of tall bookends
+    fontFamily: theme.global.type.fontFamilies.base,
+    ...contentSizes.body1(theme),
+    boxSizing: 'border-box',
+    '*, *:before, *:after': {
+      boxSizing: 'border-box',
+    },
+  }),
+  rootSmall: theme => ({
+    minHeight: fieldHeights.small,
+    ...contentSizes.caption1(theme),
+  }),
+  rootLarge: theme => ({
+    minHeight: fieldHeights.large,
+    ...contentSizes[400](theme),
+  }),
+  rootInline: {
+    display: 'inline-flex',
+  },
+  rootFilled: theme => ({
+    // optional shadow for filled appearances
+    boxShadow: theme.alias.shadow.shadow2,
+    borderRadius: borderRadius(theme), // needed for shadow
+  }),
+  rootDisabled: {
+    cursor: 'not-allowed',
+  },
+  input: theme => ({
+    flexGrow: 1,
+    border: 'none', // input itself never has a border (this is handled by inputWrapper)
+    // TODO(design) is this what "Left/Right Padding in content" means? why do we need it?
+    padding: `0 ${horizontalSpacing.xxs}`,
+    color: theme.alias.color.neutral.neutralForeground1,
```

Comment:
When filledDarker will you want neautralForeground3 here?

Reply:
No, according to what she wrote in the styling notes above it should be `neutralForeground1` for all of them.

Reply:
Just checked the design again and confirmed that the text is `neutralForeground1` in all cases